src/OrchardCore/Orchard.Environment.Shell/Builders/CompositionStrategy.cs

```diff
@@ -1,7 +1,8 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Modules;
 using Microsoft.Extensions.Logging;
 using Orchard.Environment.Extensions;
 using Orchard.Environment.Extensions.Features;
```

Comment:
Just saw that in this file a `_typeFeatureProvider` is injected but never used.
Maybe the same in other files as in `RoleStep.cs`.